packages/common/src/hooks/useDialog/useDialog.tsx

```diff
@@ -24,6 +25,7 @@ export interface ModalProps {
 
   okButton?: JSX.Element;
   width?: number;
+  sx?: SxProps<Theme>;
```

Comment:
I'm using `sx` to set the maxWidth (which is 600px by default).

Reply:
Cool, I've added the Transition as suggested, looks good.

And yeah, I'm always reluctant to add fixed widths to elements where I can avoid it ;)